benchmarks/operator_benchmark/pt/qembedding_pack_test.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Thanks for sending this out.

Can you please answer these questions?
1. What does this benchmark test actually test? Does it just measure regression to the operator over time? 
2. How does the coverage looks like w.r.t. FBGeMM kernel?
3. Can we also add 2bit ops to the benchmark?

Reply:
> What does this benchmark test actually test? Does it just measure regression to the operator over time?
Yes, this will track changes to the operator performance over time, it is tracked using AIBench and logged internally

> How does the coverage looks like w.r.t. FBGeMM kernel?
Once we make the operator use FBGEMM kernel the performance should improve and it will show up in the regression runs then.

> Can we also add 2bit ops to the benchmark?
Will do